pyartcd/pyartcd/util.py

```
@@ -1044,147 +1044,131 @@ async def reset_fail_counter(branch: str):
+                if new_count > existing_count:
+                    failures[entity_name] = failure_data
+            else:
+                failures[entity_name] = failure_data
 
         if logger and failures:
             import json
 
-            logger.info('Rebase failures for version %s: %s', version, json.dumps(failures, indent=2))
+            logger.info('Failures for pattern %s: %s', pattern, json.dumps(failures, indent=2))
 
     except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)